cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -183,6 +196,75 @@
         }
     }
 
+    protected static final Map<String, EnvVar> CUSTOM_ENV_VARS = new HashMap<>();
```

Comment:
Same as above - should these be defined in the specific models?

Reply:
The name is again a bit confusing ... These are custom labels and not custom env vars. Also, should the `Util.parseMap` be called here once instead of calling it every reconciliation again and again?